parallel-state.js

```diff
@@ -0,0 +1,43 @@
+var Nanobus = require('nanobus')
+var assert = require('assert')
+
+module.exports = Parallelstate
+
+function Parallelstate (transitions) {
+  if (!(this instanceof Parallelstate)) return new Parallelstate(transitions)
+  assert.equal(typeof transitions, 'object', 'nanostate: transitions should be type object')
+
+  this.scopes = Object.keys(transitions)
+  this.transitions = transitions
+
+  this.scopes.forEach(scope => {
+    assert.ok(transitions[scope] instanceof Nanobus, `parallel expects "${scope}" to have nanostate as transition`)
+  })
+
+  Object.defineProperty(this, 'state', {
+    get: function () {
+      return this.scopes.reduce((state, scope) => {
```

Comment:
Could we keep this as a regular function? Arrow functions are not supported yet by 15% of browsers yet. https://caniuse.com/#feat=arrow-functions. It's not a major thing, but if we can not exclude browsers out of the box, that's an easy win! :sparkles: